bundles/org.openhab.transform.rollershutterposition/src/main/java/org/openhab/transform/rollershutterposition/internal/RollerShutterPositionConstants.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.transform.rollershutterposition.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.core.thing.profiles.ProfileTypeUID;
+
+/**
+ * The {@link RollerShutterPositionConstants } class to define transform constants
+ * used across the whole binding.
+ *
+ * @author Jeff James - Initial contribution
+ */
+@NonNullByDefault
+public class RollerShutterPositionConstants {
+
+    // Profile Type UID
+    public static final ProfileTypeUID PROFILE_TYPE_UID = new ProfileTypeUID("rollershutter", "position");
```

Comment:
Other transformations seems to use `TransformationService.TRANSFORM_PROFILE_SCOPE` as first parameter (scope) and the transformation name in uppercase as second parameter (id) - in this case it would be ROLLERSHUTTERPOSITION.